captum/attr/_core/neuron/neuron_feature_ablation.py

```diff
@@ -199,7 +210,7 @@ def attribute(
             >>>                          feature_mask=feature_mask)
         """
 
-        def neuron_forward_func(*args):
+        def neuron_forward_func(*args: str):
```

Comment:
This should actually be Any, because it includes Tensors as well as any other arguments.